src/OpenTracing/Tracer.php

```diff
@@ -35,7 +35,7 @@
      */
     public function startSpan(
         $operationName,
-        SpanReference $parentReference = null,
+        Reference $parentReference = null,
         $startTimestamp = null,
         array $tags = []
     );
```

Comment:
does PHP allow arbitrary order of optional parameters (which would require syntax of param=value, param2=val)? If not, then having optional args isn't that flexible vs. passing a dictionary (although could be a lot more performant)